sunpy/visualization/wcsaxes_compat.py

```diff
@@ -104,7 +106,8 @@ def default_wcs_grid(axes):
 
 
 @u.quantity_input
-def wcsaxes_heliographic_overlay(axes, grid_spacing: u.deg = 10*u.deg, annotate=True, **kwargs):
+def wcsaxes_heliographic_overlay(axes, grid_spacing: u.deg = 10*u.deg, annotate=True,
```

Comment:
These new arguments should probably get a feature changelog entry?

Reply:
Hmm, I can see that formally speaking, but no sane user ought to be calling this function directly...  How about I put an entry under `trivial`?

Reply:
I was thinking that one could use it to overlay grids at different heights or from different times now, but maybe not. Happy to go without or with a trivial entry.

Reply:
New entry under `trivial`.

> I was thinking that one could use it to overlay grids at different heights or from different times now

Yes, that's certainly true, but I'd invariably tell a user to use `WCSAxes.get_coords_overlay()` directly instead of using this function buried under `wcsaxes_compat`.